### PR TITLE
LL-6821 - LL-6522 - CI  updates

### PR DESCRIPTION
**Pull request by valpinkman**
use https://github.com/LedgerHQ/github-actions-live-vercel/pull/1 

https://ledgerhq.atlassian.net/browse/LL-6522
https://ledgerhq.atlassian.net/browse/LL-6821

### CODE REVIEW

**Review by LFBarreto — DISMISSED** [review]

**Comment by gre** [comment]
@valpinkman please create a ticket on jira if we didn't had one. afaik it was not working so let's close for later.

**Comment by gre** [comment]
https://ledgerhq.atlassian.net/browse/LL-6522

**Comment by machard** [comment]
Capture d’écran 2021-08-10 à 11 20 58
commit hash used looks wrong

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -142,10 +142,10 @@ jobs:
       - name: post comment
         if: always() && job.status != 'cancelled'
         id: postcomment
-        uses: ./.github/actions/post-comment
+        uses: ./.github/actions/post-comment-v2
         with:
           comment: ${{ steps.comment.outputs.body }}
-          prNumber: ${{github.event.pull_request.number}}
+          sha: ${{ github.sha }}
```
Comment (gre) [inline .github/workflows/checks.yml:148]: possibly should be https://github.com/LedgerHQ/ledger-live-common/blob/master/.github/workflows/bot1.yml#L46

**Pull request merged**